lib/scaffold.dart

```diff
@@ -424,7 +420,7 @@ class BackdropScaffoldState extends State<BackdropScaffold>
 
   @override
   void dispose() {
-    if (_shouldDisposeAnimationController) _animationController.dispose();
+    if(_animationController != null) _animationController.dispose();
```

Comment:
This could be simplified further to `_animationController?.dispose();` ;)

Reply:
I just did